### DISCUSSION TITLE
How to switch off intruder's flight info and only show the ownship info?

### DISCUSSION THREAD

**Discussion by Shulu-Chen** [TUDelft-CNS-ATM/bluesky#477] [Q&A]
I have multiple aircraft, and will only focus on the target aircraft. On GUI, typically, all aircraft altitude and speed info would be along the aircraft, which is a little bit messy. Is there a way that could switch off others' info and only highlight our target aircraft?

**Comment by ProfHoekstra** [comment]
Typically we use COLOUR for that. See the COLOUR command in the Command reference.  – Jacco Hoekstra

From: Rex Chen ***@***.***>
Sent: zondag 20 augustus 2023 17:37
To: TUDelft-CNS-ATM/bluesky ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [TUDelft-CNS-ATM/bluesky] How to switch off intruder's flight info and only show the ownship info? (Discussion #477)


I have multiple aircraft, and will only focus on the target aircraft. On GUI, typically, all aircraft altitude and speed info would be along the aircraft, which is a little bit messy. Is there a way that could switch off others' info and only highlight our target aircraft?
